.github/workflows/typespec-migration-validation.yaml

```diff
@@ -0,0 +1,27 @@
+name: TypeSpec Migration Validation
+
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+
+permissions:
+  contents: read
+
+jobs:
+  typespec-migration-validation:
+    name: TypeSpec Migration Validation
+    if: contains(github.event.pull_request.labels.*.name, 'typespec-conversion-w1') || contains(github.event.pull_request.labels.*.name, 'typespec-conversion-w2')
```

Comment:
Have you tested if this works on a PR from a fork?  According to the docs, the pull_request webhook event payload is empty for PRs from forks.

https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request

Reply:
I verified this does work!  This is a great find, because it means we can condition workflows on PR labels, without needing to run code on an agent.  And it automatically re-runs on every commit, or when the label is changed.